worker/migrations/migrate-accounts-to-organisation.go

```diff
@@ -192,38 +241,42 @@ func migrateAccountsToOrganization(domain string) error {
             continue
         }
 
-        if link != "" {
+        if link == "" {
             log.Warningf("No vendor_link in manifest for %s", msg.Slug)
             continue
         }
 
-        acc := &account.Account{}
-        if err := couchdb.GetDoc(inst, consts.Accounts, msg.Account, acc); err != nil {
+        var accJson couchdb.JSONDoc
+
+        if err := couchdb.GetDoc(inst, consts.Accounts, msg.Account, &accJson); err != nil {
```

Comment:
Why use a JSONDoc instead of the account?

Reply:
Because the Account casting cannot access the full `auth` attributes, so 1st we could not access additional fields when building the cipher, and 2nd when updating the account, those extra fields were lost.

Reply:
Ok, understood. ping @nono in case you have a go-solution to use a generic field in the BasicInfo struct?

Reply:
Well, we should probably use a map for `BasicInfo` instead of a struct.